.gitmodules

```diff
@@ -37,3 +37,6 @@
 [submodule "src/extensions/default/brackets-browser-livedev"]
 	path = src/extensions/default/brackets-browser-livedev
 	url = https://github.com/humphd/brackets-browser-livedev.git
+[submodule "src/extensions/default/brackets-paste-and-indent"]
+	path = src/extensions/default/brackets-paste-and-indent
+	url = git@github.com:KeniMorri/brackets-paste-and-indent.git
```

Comment:
Let's use the original repo vs. your forked URL.  Just send him a PR and he can merge that upstream, and we'll update then.